### PR TITLE
Add Ukrainian and Russian languages

### PR DESCRIPTION
**Pull request by kuisis**

### CODE REVIEW

**Review by esau-morais — APPROVED** [review]
Thank you for contribute with Ukranian and Russian languages :smile: !!

_Note: It looks you are following the required template_

**Comment by sabesansathananthan** [comment]
Could you please Resolve Conflicts.

**Comment by kuisis** [comment]
@sabesansathananthan 
Ok, will take a look when i have time

**Comment by kuisis** [comment]
@sabesansathananthan
Conflicts resolved, please check

**Review by sabesansathananthan — APPROVED** [review]
Thank you for your PR

File: translations/README.ua.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+4. Відкрийте в браузері <http://localhost:3000>
+
+## [Демо](https://react-github-resume.vercel.app/)
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/sabesansathananthan/React-GitHub-Resume)
+
+## 📄 Ліцензія
+
+Цей проект знаходиться під ліцензією MIT License - відкрийте [LICENSE](../LICENSE) файл для подробиць
```
Comment (esau-morais) [inline translations/README.ua.md:87]: Please, you need to change `LICENCE` name to the right language

**Review by esau-morais — COMMENTED** [review]
_Note: It was noticed that you need to change:_

- [ ] License name to the right language
- [ ] Summary links to the right language

**Pull request merged**